### PR TITLE
add support for alternate ortho UD40 PCB

### PR DESCRIPTION
**Pull request by npspears**
alternate ortho PCB for UD40 sold on Checkerboards.xyz

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/checkerboards/ud40_ortho_alt/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# UD40_Ortho_Alt
+
+![UD40_Ortho_Alt](https://i.imgur.com/a/bHvpzUX.jpg)
```
Comment (fauxpark) [inline keyboards/checkerboards/ud40_ortho_alt/readme.md:3]:
```suggestion
![UD40_Ortho_Alt](https://i.imgur.com/NnGIo0O.png)
```

File: keyboards/checkerboards/ud40_ortho_alt/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+The UD40_Ortho_Alt PCB is a modified layout for the UD040. It's similar to traditional ortho but the bottom row keys are larger than 1u. 
+
+* Keyboard Maintainer: [Nasp](https://github.com/npspears)
+* Hardware Supported: UD40
+* Hardware Availability: MyKeyboard.EU
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ud40_ortho_alt:default
```
Comment (fauxpark) [inline keyboards/checkerboards/ud40_ortho_alt/readme.md:13]:
```suggestion
    make checkerboards/ud40_ortho_alt:default
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by npspears — COMMENTED** [review]
suggestions committed

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**